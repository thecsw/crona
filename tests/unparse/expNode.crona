// Handles:
// CallExpNode
// CallStmtNode
// HavocNode
// StrLitNode
// LValNode->IndexNode
// ArrayTypeNode
// ByteTypeNode

fn : void (var1 : string, var2 : byte array[10]) {
    var1 = "This is a string";
    var2[2] = 1;
    fn();
}