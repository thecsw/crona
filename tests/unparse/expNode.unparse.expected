fn:void(var1:byte array[0], var2:byte array[10]){
	var1 = "This is a string";
	var2[2] = 1;
	fn();
}
